config-heroku.ini

```diff
@@ -1,12 +1,12 @@
 [General]
 # Instance name can be used to distinguish between different instances using backend api.getInstanceName()
-instanceName="heroku"
+instanceName="heroku-dafmdev"
 
 # set to true to allow using Trilium without authentication (makes sense for server build only, desktop build doesn't need password)
-noAuthentication=false
+noAuthentication=true
 
 # set to true to disable backups (e.g. because of limited space on server)
-noBackup=true
```

Comment:
I don't think it makes sense to backup at heroku because I currently haven't found a way to get the backup data back.